bentoml/_internal/frameworks/transformers.py

```diff
@@ -1,86 +1,41 @@
-import re
+from __future__ import annotations
+
+import os
 import typing as t
 import logging
 import importlib.util
 from typing import TYPE_CHECKING
-from importlib import import_module
 
-from simple_di import inject
-from simple_di import Provide
+import pydantic
```

Comment:
Is it possible that we use `attrs` instead of `pydantic`. I believe that `pydantic` is a very heavy library to import, and for the cases of `_Schema` a simple attrs class as a dataclass would work completely well and fast.

Reply:
currently pydantic is not a dependency of BentoML, would be great to replace it with attrs